index.js

```diff
@@ -66,15 +66,15 @@ function paginate(query, options, callback) {
   return Promise.all(promises).then((data) => {
     let result = {
       docs: data.docs,
-      total: data.count,
```

Comment:
This should stay as is (_data.count_). Because property count is result of **@estimatedDocumentCount** method.